src/Compositor.cpp

```diff
@@ -2212,10 +2212,27 @@ void CCompositor::warpCursorTo(const Vector2D& pos, bool force) {
     // warpCursorTo should only be used for warps that
     // should be disabled with no_cursor_warps
 
-    static auto* const PNOWARPS = &g_pConfigManager->getConfigValuePtr("general:no_cursor_warps")->intValue;
+    static auto* const PNOWARPS        = &g_pConfigManager->getConfigValuePtr("general:no_cursor_warps")->intValue;
+    static auto* const PNOWARPSFOCUSED = &g_pConfigManager->getConfigValuePtr("general:no_cursor_warps_if_focused")->intValue;
+    static auto* const PGAPSIN         = &g_pConfigManager->getConfigValuePtr("general:gaps_in")->intValue;
 
-    if (*PNOWARPS && !force)
```

Comment:
I dont think this belongs here. I'd rather just not warp to anything if monitor stays the same in changeWorkspace for now, and figure out a better method later.

Reply:
It is from https://github.com/hyprwm/Hyprland/commit/61c9e50bcd6c51e026203f194bc186eda3c44a38